### PR TITLE
docs: add new community plugins project area

### PR DESCRIPTION
**Pull request by vinzscam**
## Hey, I just made a Pull Request!

Let's add a new project area for the [Community plugins repository](https://github.com/backstage/community-plugins).

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: OWNERS.md [hunk 1]
```diff
@@ -108,6 +108,20 @@ Scope: The TechDocs plugin and related tooling
 
 These incubating project areas have shared ownership with @backstage/maintainers.
 
+### Community Plugins
+
+Team: @backstage/community-plugins-maintainers
+
+Scope: Tooling related to the Backstage [Community Plugins repository](https://github.com/backstage/community-plugins)
```
Comment (benjdlambert) [inline OWNERS.md:115]: I wonder if it makes sense to update the scope for this for now to something a little wider. 

```suggestion
Scope: Tooling and Community Repo Maintainers for the Backstage [Community Plugins repository](https://github.com/backstage/community-plugins)
```

Comment (benjdlambert) [reply OWNERS.md:115]: Should probably link out to some of the governance to what is a Reviewer / Repo Maintainer / Plugin Maintainer in the community docs

**Review by benjdlambert — APPROVED** [review]

File: OWNERS.md [hunk 2]
```diff
@@ -108,6 +108,20 @@ Scope: The TechDocs plugin and related tooling
...
+Team: @backstage/community-plugins-maintainers
+
+Scope: Tooling and Community Repo Maintainers for the Backstage [Community Plugins repository](https://github.com/backstage/community-plugins)
+
+| Name                 | Organization | GitHub                                      | Discord      |
+| -------------------- | ------------ | ------------------------------------------- | ------------ |
+| Bethany Griggs       | Red Hat      | [BethGriggs](https://github.com/BethGriggs) | `bethgriggs` |
+| Tomas Kral           | Red Hat      | [kadel](https://github.com/kadel)           | `tomkral`    |
+| André Wanlin         | Spotify      | [awanlin](https://github.com/awanlin)       | `ahhhndre`   |
+| Philipp Hugenroth    | Spotify      | [tudi2d](https://github.com/tudi2d)         | `tudi2d`     |
```
Comment (Rugvip) [inline OWNERS.md:122]: @BethGriggs @kadel @awanlin @tudi2d could each of you approve this PR if this all looks good? 🙏

**Review by tudi2d — APPROVED** [review]

**Review by awanlin — APPROVED** [review]
🚀

**Review by kadel — APPROVED** [review]

**Comment by benjdlambert** [comment]
Wahey :tada: we're all signed off! Welcome!

**Pull request merged**